packages/cardano-services/src/Asset/AssetHttpService.ts

```diff
@@ -0,0 +1,55 @@
+import * as OpenApiValidator from 'express-openapi-validator';
+import { CardanoTokenRegistryProvider } from './CardanoTokenRegistryProvider';
+import { HttpService } from '../Http';
+import { Logger } from 'ts-log';
+import { ServiceNames } from '../Program';
+import { TokenMetadataProvider } from './types';
+import { providerHandler } from '../util';
+import express from 'express';
+import path from 'path';
+
+/**
+ * Dependencies that are need to create AssetHttpService
+ */
+export interface AssetHttpServiceDependencies {
+  /**
+   * The logger object
+   */
+  logger: Logger;
+
+  /**
+   * The token metadata provider. Default: CardanoTokenRegistry instance
+   */
+  tokenMetadataProvider?: TokenMetadataProvider;
```

Comment:
```suggestion
  assetProvider?: AssetProvider;
```
Token metadata is just one part of the asset provider

Reply:
Yep, AssetProvider is the only external interface here. It has NftMetadataService and TokenMetadataService as internal dependencies.

Reply:
cd7458e should resolve this